omero/sysadmins/unix/walkthrough/walkthrough_debian9.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e -u -x
+source settings.env
+
+#start-step01: As root, install dependencies
+
+apt-get update
+
+# installed for convenience
+apt-get -y install unzip wget bc
+
+# install Java
+apt-get -y install openjdk-8-jre-headless
+
+# install dependencies
+
+apt-get -y install \
+	python-{pip,pillow,numpy,scipy,tables,virtualenv,yaml,jinja2}
+
+pip install --upgrade pip
+# install Ice
+#start-recommended-ice
+apt-get -y install libssl-dev libbz2-dev libmcpp-dev libdb++-dev libdb-dev libdb-java
+apt-get -y install zeroc-ice-all-runtime
+pip install "zeroc-ice>3.5,<3.7"
+#end-recommended-ice
+
+
+# install Postgres
+apt-get -y install postgresql
+service postgresql start
```

Comment:
The CentOS7 omero-install test uses a systemd-enabled image: https://github.com/ome/omero-install/blob/develop/linux/test/centos7_nginx/Dockerfile#L5 (this is one we wrote ourselves, though CentOS have since released their own systemd Docker image). Could check if Debian has something similar?